h/auth/__init__.py

```diff
@@ -41,6 +44,15 @@ def includeme(config):
     global DEFAULT_POLICY
     global WEBSOCKET_POLICY
 
+    config.register_service_factory('.services.auth_ticket_service_factory',
+                                    iface=pyramid_authsanity.interfaces.IAuthService)
+
+    # Set up authsanity
```

Comment:
Might be worth moving this comment up a couple of lines, to indicate that the `auth_ticket_service_factory` is part of the configuration.